### PR TITLE
RULES.md: initial document stating the LibreELEC repository usage rules.

### PR DESCRIPTION
**Pull request by lrusak**
I would like the entire team to comment on this

This is just an initial document, it can be amended in the future to include any other rules that we need to define about the LibreELEC project.

### CODE REVIEW

**Comment by lrusak** [comment]
Would like to see a few more comments here from the Team, if we think it's ready let me know and I'll squash

File: RULES.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## Package Version Bumps
+
+If a package version is broken it must be signified with a comment stating what is broken and possibly a linked bug report.
+```
+# version 1.23 is broken on some systems, see https://bug.report.com/123456
+```
+
+#### LibreELEC Testing Window
+
+Anything listed in the [Major Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#major-packages) section **MUST** be submitted as a PR and reviewed.
```
Comment (MilhouseVH) [inline RULES.md]: Nit: You don't need `https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md` on the target, so `[Major Packages](#major-packages)` should work.

File: RULES.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Anything listed in the [Major Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#major-packages) section will **NOT** be bumped unless there is majority support for it.
+
+Anything listed in the [Minor Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#minor-packages) section may be bumped
+
+#### LibreELEC Release Window
+
+Once an LibreELEC release has entered beta anything in the [Major Packages](https://github.com/LibreELEC/LibreELEC.tv/blob/master/RULES.md#major-packages) section will **NOT** be bumped with the exception of Kodi as it progresses to it's final release.
```
Comment (MilhouseVH) [inline RULES.md]: Nit: s/an/a/ - `Once a LibreELEC...`?

File: RULES.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+```
+kodi: add patch to fix something
+```
+```
+systemd: update to 229
+```
+
+## Pull Requests
+
+Merging can only be done so after the pull request has been reviewed and signed off by the submitter and a team member
```
Comment (MilhouseVH) [inline RULES.md]: "done so after" => "done after"

File: RULES.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Any fixes to outstanding issues must be submitted as a PR **and** be tested.
+
+## Reverts
+
+Any reverts must be submitted as a PR **and** provide information about why the revert needs to occur.
+
+## Trust
+
+These master are of course only practical if we can trust each other to use them accordingly.
```
Comment (MilhouseVH) [inline RULES.md]: "These master" - not sure what you're saying here

**Comment by MilhouseVH** [comment]
Other than the nits, it looks good.

My only reservation is this line:

`Once an LibreELEC release has entered beta anything in the Major Packages section will NOT be bumped with the exception of Kodi as it progresses to it's final release.`

While I agree in general, what if we find a show-stopper bug in say, `connman`? Ideally we'd patch the package and fix the bug, but in the worst case we may need to bump the package (or even revert a bump). The language here suggests we won't allow ourselves to do either. Maybe we should add a clause to cater for exceptional circumstances, rather than paint ourselves into a corner.

**Comment by WeinerRinkler** [comment]
Sorry for offtopic comment, but I do not know where I can ask, your irc channel is invitation only.
I just found this fork and I wonder whats the goals of this fork. Can you answer me this question?
Greetings

**Comment by chewitt** [comment]
@WeinerRinkler - website and forums will be up shortly to explain, thanks :)

**Comment by lrusak** [comment]
Updated with @MilhouseVH changes.
Please give a final review.

**Pull request closed without merging**